### DISCUSSION TITLE
How to contribute to stockfish

### DISCUSSION THREAD

**Discussion by chriszeng1010** [official-stockfish/Stockfish#5689] [General]
Hey, newbie here.

I'd like to contribute to stock fish code base. 

I looked through the issue list, but it seems like the issues are mostly suggestions or feature requests. 

Is there a place for someone like me to contribute and ramp up on the source code?

Thanks a bunch.

**Comment by cj5716** [comment]
You can look at

https://github.com/official-stockfish/Stockfish/wiki/Developers#participating-in-the-project

In general most development happens in search selectivity or move ordering in recent times
